fcrepo-webapp/src/main/webapp/index.html

```diff
@@ -15,30 +15,32 @@
             padding-top: 70px;
             padding-bottom: 70px;
         }
-
+        article p {
+          padding: 10px 0;
+        }
     </style>
 </head>
 <body>
-<div class="container">
-<div class="col-md-12">
-<div class="jumbotron">
-<a href="rest/" class="thumbnail" style=""><img src="images/fedora_logo_10in.png" /></a>
-<p>This is the exciting stub page for your Fedora ${project.version} repository that we have placed
-here.</p>
+<article class="container">
+  <div class="col-md-12">
+    <div class="jumbotron">
+      <a href="rest/" class="thumbnail" style=""><img src="images/fedora_logo_10in.png" /></a>
+      <p>Fedora is a robust, modular, open source repository system for the management and dissemination of digital content. It is especially suited for digital libraries and archives, both
+for access and preservation. It is also used to provide specialized access to very large and complex digital collections of historic and cultural materials as well as scientific
+data.</p>
 
-<p>You probably want to visit something a little more interesting, such as:       </p>
-<a class="btn btn-primary btn-lg" href="rest/">the Fedora REST API endpoint</a>
-<p><br/></p>
-<p>Fedora is maintained by a community of volunteers. <a href="http://duraspace.org/register-repository" target="_duraspace">Register your repository</a> at <a href="http://duraspace.org/" target="_duraspace">DuraSpace</a> in order to facilitate information sharing and collaboration within the community.</p>
-</div>
-</div>
-</div>
+      <p class="text-center"><a class="btn btn-primary btn-lg" href="rest/">Fedora REST API endpoint</a></p>
+      <p>Fedora is maintained by a community of volunteers. <a href="http://duraspace.org/register-repository" target="_duraspace">Register your repository</a> at <a href="http://duraspace.org/" target="_duraspace">DuraSpace</a> in order to facilitate information sharing and collaboration within the community.</p>
+    </div>
+  </div>
+</article>
 
 <nav class="navbar navbar-default navbar-fixed-bottom" role="navigation">
     <div class="container">
         <ul class="nav navbar-nav">
             <li><a href="https://jira.duraspace.org/browse/FCREPO">Issue Tracker</a></li>
             <li><a href="https://wiki.duraspace.org/display/FF">Wiki</a></li>
+            <li><a href="https://wiki.duraspace.org/display/FEDORA4x/RESTful+HTTP+API">REST API Documentation</a></li>
```

Comment:
Please let me know whatever new work you are making for me, once you know what it is.

The obvious (although not obviously correct) way to move the value would be via [Maven Resource action](https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html).